### DISCUSSION TITLE
Challenge/NFT for the StableCoin part

### DISCUSSION THREAD

**Discussion by devblixt** [Cyfrin/foundry-full-course-cu#78] [Q&A] [answered]
Hey folks, 

I heard on Twitter that you're supposed to mint an NFT for each lesson of the course. I have some prior experience in solidity and I just wanted to do the stablecoin part, so I completed it. However, I did not find any challenge/NFT to be minted in the section. Did I miss something? 

Have a great day!

**Comment by alymurtazamemon** [comment, marked as answer]
@devblixt here are the links;
1. Arbitrum = https://arbiscan.io/address/0x3DbBF2F9AcFB9Aac8E0b31563dd75a2D69148D64
2. Sepolia testnet = https://sepolia.etherscan.io/address/0xe5760847db2f10A74Fc575B4803df5fe129811C1

**Reply by devblixt** [reply]
Thank you, I've solved it as well!
